src/main/java/com/yelp/nrtsearch/server/grpc/LuceneServerClient.java

```diff
@@ -112,71 +113,8 @@ public void createIndex(
     try {
       response = blockingStub.createIndex(requestBuilder.build());
     } catch (StatusRuntimeException e) {
-      logger.warn("RPC failed: {}", e.getStatus());
-      return;
-    }
-    logger.info("Server returned : " + response.getResponse());
-  }
-
-  public void liveSettings(
-      String indexName,
-      double maxRefreshSec,
-      double minRefreshSec,
-      double maxSearcherAgeSec,
-      double indexRamBufferSizeMB,
-      int addDocumentsMaxBufferLen,
-      int sliceMaxDocs,
-      int sliceMaxSegments,
-      int virtualShards,
-      int maxMergedSegmentMB,
-      int segmentsPerTier,
-      double defaultSearchTimeoutSec,
-      int defaultSearchTimeoutCheckEvery,
-      int defaultTerminateAfter) {
-    logger.info(
-        String.format(
-            "will try to update liveSettings for indexName: %s, "
-                + "maxRefreshSec: %s, minRefreshSec: %s, maxSearcherAgeSec: %s, "
-                + "indexRamBufferSizeMB: %s, addDocumentsMaxBufferLen: %s, sliceMaxDocs: %s, "
-                + "sliceMaxSegments: %s, virtualShards: %s, maxMergedSegmentMB: %s, segmentsPerTier: %s, "
-                + "defaultSearchTimeoutSec: %s, defaultSearchTimeoutCheckEvery: %s, defaultTerminateAfter: %s ",
-            indexName,
-            maxRefreshSec,
-            minRefreshSec,
-            maxSearcherAgeSec,
-            indexRamBufferSizeMB,
-            addDocumentsMaxBufferLen,
-            sliceMaxDocs,
-            sliceMaxSegments,
-            virtualShards,
-            maxMergedSegmentMB,
-            segmentsPerTier,
-            defaultSearchTimeoutSec,
-            defaultSearchTimeoutCheckEvery,
-            defaultTerminateAfter));
-    LiveSettingsRequest request =
-        LiveSettingsRequest.newBuilder()
-            .setIndexName(indexName)
-            .setMaxRefreshSec(maxRefreshSec)
-            .setMinRefreshSec(minRefreshSec)
-            .setMaxSearcherAgeSec(maxSearcherAgeSec)
-            .setIndexRamBufferSizeMB(indexRamBufferSizeMB)
-            .setAddDocumentsMaxBufferLen(addDocumentsMaxBufferLen)
-            .setSliceMaxDocs(sliceMaxDocs)
-            .setSliceMaxSegments(sliceMaxSegments)
-            .setVirtualShards(virtualShards)
-            .setMaxMergedSegmentMB(maxMergedSegmentMB)
-            .setSegmentsPerTier(segmentsPerTier)
-            .setDefaultSearchTimeoutSec(defaultSearchTimeoutSec)
-            .setDefaultSearchTimeoutCheckEvery(defaultSearchTimeoutCheckEvery)
-            .setDefaultTerminateAfter(defaultTerminateAfter)
-            .build();
-    LiveSettingsResponse response;
-    try {
-      response = blockingStub.liveSettings(request);
-    } catch (StatusRuntimeException e) {
-      logger.warn("RPC failed: {}", e.getStatus());
-      return;
+      logger.error("Unable to create index {}", indexName, e);
+      throw e;
```

Comment:
Is there a reason the exception is being provided to the logging call?

Reply:
It was leftover from some trial-and-error, removed now. Thanks for catching!